### PR TITLE
Remove gh-pages folder related of an no more existing branch

### PR DESCRIPTION
**Pull request by sbrunner**
[https://github.com/camptocamp/c2cgeoportal/issues/1937]

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -159,6 +164,9 @@ gh-pages: .build/ngeo-$(GITHUB_USERNAME)-gh-pages \
 	cd $<; git merge --ff-only origin/gh-pages
 	cd $<; git rm --ignore-unmatch -r --quiet --force $(GIT_BRANCH)
 	cd $<; git clean --force -d
+	.build/python-venv/bin/python buildtools/cleanup-ghpages.py $<
+	cd $<; git add -A
+	cd $<; git commit -m 'Cleanup GitHub pages'
```
Comment (pgiraud) [inline Makefile]: What happens if the GITHUB_USERNAME is not camptocamp?

Comment (sbrunner) [inline Makefile]: Remove everything ... :-)

Comment (sbrunner) [inline Makefile]: I will fix it :-)

Comment (sbrunner) [inline Makefile]: Fixed :-)

**Comment by pgiraud** [comment]
Thanks. Please merge.

**Pull request merged**